tests/browser-tests/AnchorLinks.browser.ts

```diff
@@ -0,0 +1,27 @@
+import { BASE_URL } from '../../test-utils/baseUrl';
+import { ClientFunction, Selector, t } from 'testcafe';
+
+fixture`AnchorLinks`.page`${BASE_URL}`;
+
+const getWindowScollY = ClientFunction(() => window.scrollY);
+
+test('No anchor, defaults to top of page', async () => {
+    await t.navigateTo('/author-guide/journal-policies');
+    await t.wait(500); // Wait, so the useScrollToAnchor hook has run.
+    await t.expect(getWindowScollY()).eql(0);
+});
+
+test('With anchor, scrolls the section into view', async () => {
+    await t.navigateTo('/author-guide/journal-policies#name-change');
```

Comment:
Theres a lot of repetition in these checks 
- go to page
- wait for functionality
- expect scroll offset

Could this be added to the [StaticPage page object](https://github.com/libero/reviewer-client/blob/cc398819519a070f4daa495d08dcfd2061b3a87e/tests/page-objects/StaticPage.ts) as some form of util function?